types/cheerio/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Cheerio v0.22.0
+// Type definitions for Cheerio v0.22.1
```

Comment:
Only update the version if you're targeting a new version of the library

Reply:
I removed this change.